canary/package.json

```diff
@@ -18,7 +18,7 @@
     "angular": "yarn angularcli",
     "angular:install": "yarn angularcli:install",
     "cra": "yarn --cwd apps/react/cra build ",
-    "cra:install": "yarn --cwd apps/react/cra install",
+    "cra:install": "echo 'NOTE: Testing using npm install instead of yarn' && npm install apps/react/cra install",
```

Comment:
This works - it looks like it installs on the root canary folder and the `apps/react/cra` folder.

You could also use `npm --prefix apps/react/cra install` which is more precise

Reply:
Good idea, I will update to use the --prefix

Reply:
Updated